secure_headers.gemspec

```diff
@@ -15,5 +15,5 @@ Gem::Specification.new do |gem|
   gem.test_files    = gem.files.grep(%r{^(test|spec|features)/})
   gem.require_paths = ["lib"]
   gem.add_development_dependency "rake"
-  gem.add_dependency "useragent"
+  gem.add_dependency "useragent", "0.16.8"
```

Comment:
Not a fan of pinning to a specific version, as it makes it a big pain to fix when there's a security issue or something with a gem. Can't we just do `"~> 0.16.8"` or something if that's the minimum version we actually need?